src/NCI.OCPL.Api.CTSListingPages/Services/ESListingInfoQueryService.cs

```diff
@@ -1,5 +1,6 @@
 using System;
```

Comment:
Missing a test for the Elasticsearch connection failing. (e.g. returning only a partial response.)

Reply:
Added!